tests/playwright-test/reporter-github.spec.ts

```diff
@@ -73,4 +73,22 @@ test('print GitHub annotations for slow tests', async ({ runInlineTest }) => {
   expect(text).toContain('::warning title=Slow Test,file=a.test.js::a.test.js took');
   expect(text).toContain('::notice title=🎭 Playwright Run Summary::  1 passed');
   expect(result.exitCode).toBe(0);
-});
\ No newline at end of file
+});
+
+test('print GitHub annotations for global error', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'a.test.js': `
+      const test = pwt.test.extend({
+        w: [async ({}, use) => {
+          await use();
+          throw new Error('Oh my!');
+        }, { scope: 'worker' }],
+      });
+      test('passes but...', ({w}) => {
+      });
+    `,
+  }, { reporter: 'github' });
+  const text = stripAscii(result.output);
+  expect(text).toContain('::error ::%0AError: Oh my!%0A%0A');
```

Comment:
would a trim() help there? 0a seems new line

Reply:
This is actually the format of github annotations - `%0A` instead of new lines.